content/_releases/22/28.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Product Update - v22.28
+description: Details of the product update v22.28 from 14th of July 2022.
+layout: rel
+version: "22.28"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-07-14
+redirect_from:
+  - /releases/22.28.html
+---
+
+## Improvements and Updates
+
+### Per-tenant frontend styles feature removed
+
+We've removed the feature that builds per-tenant styles and applied them based on `tenant.cssEnabled` and `tenant.code property`. We also removed all tenant styles from our codebase. Please use `tenant.customStylesheets` instead.
+
+> PLease visit our [API documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#tenants) for more information.
+
+###  Mailchimp is no longer supported
+
+Since Mailchimp is no longer supported we have removed everything related to Mailchimp from platform. It just removes redundant code from the code base and does not affect the [Mailchimp component](/components/mailchimp/).
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+###   Add Ability to Set Per-Service Custom Ingress Annotations
+
+To increase flexibility in ingress configurations for different hosts we made a **breaking change**. We have changed the structure of the model properties `tenant.ingressConfig.annotations` and `helm.Values.global.services.handmaiden.annotations`. Please be **aware** that upgrading to elastic.io Tenants to this latest version requires the small changes described below:
+
+  * `tenant.ingress_config.annotations` format changed, **unset it via API before upgrade platform with current release** and set to a new format after. This parameter must be reset via API call before updating to the latest version. Pleas use this API call:
```

Comment:
format changed**.**
DELETE: **unset it via API before upgrade platform with current release** and set to a new format after.

Pleas**e**